cli/src/commands/init.ts

```diff
@@ -23,17 +26,27 @@ export const handler = async (argv: Arguments<Options>): Promise<void> => {
   }
 
   console.log("initing PSP...");
-  const anchorPath = path.resolve(__dirname, "../../bin/light-anchor");
+  const cargoGeneratePath = path.resolve(__dirname, "../../bin/cargo-generate");
   const dirPath = path.resolve(__dirname, "../../bin/");
 
-  await downloadFileIfNotExists({
-    filePath: anchorPath,
+  await downloadCargoGenerateIfNotExists({
+    localFilePath: cargoGeneratePath,
     dirPath,
-    repoName: "anchor",
-    fileName: "light-anchor",
   });
 
-  execSync(`${anchorPath} init-psp ${name}`);
+  const circomName = snakeCase(name);
+  const rustName = snakeCase(name);
+  execSync(
+    `${cargoGeneratePath} generate \
+    --git https://github.com/Lightprotocol/psp-template \
+    --branch vadorovsky/cargo-generate-template \
```

Comment:
We need to remove this line after merging

https://github.com/Lightprotocol/psp-template/pull/1